docs/content/test_runner/Metrics.md

```diff
@@ -4,7 +4,7 @@
 
 Metrics available at:
 
-`$namespace.testrunner.$buildId.$instrumentationConfigName.`
+`$namespace.testrunner.$buildId.$projectName.$instrumentationConfigName.`
```

Comment:
It may inconvenient to change a variable part of metric.

What we have before:
`groupByNodes($namespace.testrunner.$buildId.$instrumentationConfigName...., <positon of instrumentationConfigName>)` - metrics grouped by configurations. 

What will we get:
`groupByNodes($namespace.testrunner.$buildId.$projectName.$instrumentationConfigName...., <position of projectName>)` - metrics grouped by projects? No, because here we have already `instrumentationConfigName` serieses with previous layout.

Not a big deal, can be filtered manually. Just wanted to mention. 
I don't know best practices on how to make breaking changes in metrics.

Reply:
yep, i agree, but also don't know how to handle such changes correctly

Reply:
Here is an alternative solution - tags

https://graphite.readthedocs.io/en/latest/tags.html
See an example in PER-1325.

It looks easier to extend but preserves original values without details for trends.